Patterns/English/English-DateTime.yaml

```diff
@@ -1,1085 +1,1085 @@
----
-TillRegex: !nestedRegex
-  def: (?<till>\b(to|till|til|until|thru|through)\b|{BaseDateTime.RangeConnectorSymbolRegex})
-  references: [ BaseDateTime.RangeConnectorSymbolRegex ]
-RangeConnectorRegex : !nestedRegex

```

Comment:
Please be careful no not commit changes that are only line breaks. This makes it much harder to review a PR. Can you please revert them in this file?